File: Iniciante/1016 - Distância/TypeScript/1016.ts
import * as fs from "fs";

function main(): void {
  const input: string = fs.readFileSync("/dev/stdin", "utf8");
  const distance: number = parseInt(input);
  const time: number = calculateTime(distance);
  console.log(`${time} minutos`);
}

function calculateTime(distance: number): number {
    const TIME_PER_KILOMETER = 2;
    return distance * TIME_PER_KILOMETER;
}

main();

File: Iniciante/1016 - Distância/Python/1016.py
def main():
    distance = int(input())
    time = calculate_time(distance)
    print(f"{time} minutos")


def calculate_time(distance):
    TIME_PER_KILOMETER = 2
    return distance * TIME_PER_KILOMETER


if __name__ == "__main__":
    main()
